packages/terra-form-field/src/Field.scss

```diff
@@ -0,0 +1,78 @@
+:local {
+  .field {
+    border: 0;
+    margin-bottom: var(--terra-form-field-margin-bottom, 1.428em);
+    max-width: 1020px;
+    padding: 0;
+  }
+
+  .field-inline {
+    display: inline-block;
+    margin-right: var(--terra-form-field-inline-margin-right, 0.7143em);
+    vertical-align: top;
+  }
+
+  .label-group {
+    font-family: var(--terra-form-field-label-font-family, inherit);
```

Comment:
font-family can be removed from this instance. The label font family should be the same font family as what is set on the html element and the label-group div / label element will inherit that font through the cascade without us needing to set any styles.

Reply:
I had the change locally & didn't have it pushed. Should be good to go now.